main.yml

```diff
@@ -373,6 +373,9 @@ openshift:
       - id: subscriptions
         title: Subscriptions
         group: openshift
+      - id: overview
```

Comment:
you probably want to put this above "Clusters" if we're going to make it the default view

Reply:
As a first step it wont be the default view - we will increment in staging :smile: